### PR TITLE
Created grid component.

### PR DESCRIPTION
**Pull request by j24m**
Created grid component. kindly review
Deploy link : https://eclectic-cheesecake-7b54b2.netlify.app/components/grid.html

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Impressive UI and clean code 

Few minor bugs pointing: 

- In Toast component, there is no display of grid in the list
- When it is of grids page, highlighted link is text utils 

Please fix above changes 

Rest is all good for approval ✅

File: Components/grid.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                        <li>Grid</li>
+                    </a>
+                </ul>
+        </div>
+        <div class="documentation">
+            <h1 class="documentation-component-main-heading">Grid</h1>
+            <h2 class="component-info">50-50 Grid :</h2>
+            <div class="grid-container">
+                <div class="grid-50-50">
+                    <img class="img-responsive" src="https://picsum.photos/500/400" alt="">
```
Comment (mehulsatardekar) [inline Components/grid.html:88]: well you can add some text inside alt tag dont leave alt tag empty

File: Components/grid.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                </div>
+                <div class="code-snippet">
+                    <script src="https://gist.github.com/j24m/07cceed710deea2db14253e56299b90b.js"></script>
+                </div>
+ 
+            <h2 class="component-info">Three equal grids :</h2>
+
+                <div class="grid-equal">
+                    <img class="img-responsive" src="https://picsum.photos/500/400" alt="">
+                    <img class="img-responsive" src="https://picsum.photos/500/400" alt="">
```
Comment (mehulsatardekar) [inline Components/grid.html:108]: here also add alt text inside alt tag

**Review by mehulsatardekar — COMMENTED** [review]
The Rest thing looks good and ready to merge

**Pull request merged**